src/main/java/it/gov/pagopa/payhub/activities/dto/classifications/TransferSemanticKeyWithOutComeCodeDTO.java

```diff
@@ -0,0 +1,14 @@
+package it.gov.pagopa.payhub.activities.dto.classifications;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+public class TransferSemanticKeyWithOutComeCodeDTO extends TransferSemanticKeyDTO {
```

Comment:
let's use a more generic name
```suggestion
public class PaymentsReportingTransferDTO extends TransferSemanticKeyDTO {
```